test/__init__.py

```diff
@@ -812,6 +811,12 @@ def require_no_api_version(self, func):
             func=func,
         )
 
+    def require_sync(self, func):
+        """Run a test only if using the synchronous API."""
+        return self._require(
+            lambda: _IS_SYNC, "This test only works with the synchronous API", func=func
```

Comment:
Since _IS_SYNC is defined as true, would the lambda ever fail?

Reply:
This file is generated from `test/asynchronous/__init__.py`, it always passes for a synchronous test, as intended.